app/models/school_workforce_census.rb

```diff
@@ -2,21 +2,7 @@ class SchoolWorkforceCensus < ApplicationRecord
   self.table_name = "school_workforce_censuses"
 
   SUBJECT_ATTRIBUTES = [
-    :subject_1,
-    :subject_2,
-    :subject_3,
-    :subject_4,
-    :subject_5,
-    :subject_6,
-    :subject_7,
-    :subject_8,
-    :subject_9,
-    :subject_10,
-    :subject_11,
-    :subject_12,
-    :subject_13,
-    :subject_14,
-    :subject_15
+    :subject_description_sfr
```

Comment:
❓ Presumably, this `SUBJECT_ATTRIBUTES` is now semantically useless. Also, am I right in saying that the `subjects` method will need to be changed / removed? 

I'd imagine we'd want to retrieve all subjects for the selected teacher (found by URN), so we'd need a relationship or a join with this table by URN and retrieve all subjects from the returned rows?